alacritty_terminal/src/ansi.rs

```diff
@@ -215,6 +215,9 @@ pub trait Handler {
     /// Linefeed
     fn linefeed(&mut self) {}
 
+    /// Linefeed and carriage return
+    fn linefeed_carriage_return(&mut self) {}
```

Comment:
What is the purpose of adding this function? I mean linefeed should be fine, right?

Reply:
In `input`, every linewrap is a linefeed followed by a carriage return, which without this updates the damage rect twice.

I thought that was silly, so I added this combined function.